tests/ha/priority_fencing_delay.pm

```diff
@@ -0,0 +1,72 @@
+# SUSE's openQA tests
+#
+# Copyright (c) 2021 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test the priority fencing delay feature
+# The node with the master resource must always win the fencing match
+# Maintainer: Julien Adamek <jadamek@suse.com>
+
+use base 'opensusebasetest';
+use strict;
+use warnings;
+use testapi;
+use lockapi;
+use hacluster;
+
+sub stonith_iptables {
+    my ($self, $count, $cluster) = @_;
+    my $partner_ip = is_node(1) ? get_ip(choose_node(2)) : get_ip(choose_node(1));
+
+    while ($count ne 0) {
+        script_run "iptables -A INPUT -s $partner_ip -j DROP; iptables -A OUTPUT -d $partner_ip -j DROP";
+        if (is_node(1)) {
+            # Wait for the stonith match, then flush the rules for the next test
+            sleep 20;
```

Comment:
Is it a good idea to add a `scale_timeout` command here? And is this really needed? I mean checking for the status before doing the next command might be an option?

Clearly I think it's better to not use `sleep` command like this. I know I used some previously but I know try to remove them. as it's more a cause of sporadic issues at the end...

Reply:
I agree, I will try to do something with this output:
```
crm_mon -s
CLUSTER WARN: offline node: 1c004
```

Reply:
I added the same comment below. If you use `script_run` or `assert_script_run` to check, there's no need to scale the timeout though.